_posts/2021-04-14-accessing-the-relay-store.markdown

```diff
@@ -0,0 +1,260 @@
+---
+layout: epic
+title: "A Cool Way to Access the Relay Store"
+date: 2021-04-14
+categories: [relay, graphql, react, redis, tooling]
+author: anna
+---
+
+Need to access the Relay store without using a mutation? Relay's `commitLocalUpdate` allows you to modify the relay
+store from inside any relay component. The solution is particularly well suited to situations that just require
+client-side updates and do not involved changes to server-side data.
+
+<!-- more -->
+
+```js
+commitLocalUpdate(relay.environment, (store) => {
+  const parentRecord = store.get(parentID)
+
+  if (parentRecord) {
+    const artworksConnection = ConnectionHandler.getConnection(
+      parentRecord,
+      "ArtworksMissingMetadata_partner_artworksConnection"
+    )
+    if (artworksConnection) {
+      ConnectionHandler.deleteNode(artworksConnection, id)
+    }
+  }
+})
+```
```

Comment:
It's cool that there's a TLDR solution right here at the top, but after giving it a full read I think that that this can be removed in favor of the larger blog post which goes into the details nicely with a solution at the bottom. (If anything, we could include a tldr-esque link that goes right to it, though not sure if its necessary.)

Reply:
I replaced with a more general intro